src/components/phone/ContactCard.vue

```diff
@@ -66,8 +67,6 @@ export default {
     name: VueTypes.string,
     mobile: VueTypes.string,
     profileSrc: VueTypes.string,
-    available: VueTypes.bool,
-    onToggle: VueTypes.func,
```

Comment:
you removed my added props for toggling caller state availability and providing an onToggle callback